src/components/SlateEditor/plugins/embed/RelatedArticleBox.jsx

```diff
@@ -37,12 +37,15 @@ class RelatedArticleBox extends React.Component {
     this.fetchRelated = this.fetchRelated.bind(this);
     this.onInsertBlock = this.onInsertBlock.bind(this);
     this.updateNodeAttributes = this.updateNodeAttributes.bind(this);
+    this.openEditMode = this.openEditMode.bind(this);
   }
 
   componentDidMount() {
     const { embed } = this.props;
     if (embed['article-ids']) {
-      embed['article-ids'].split(',').map(it => this.fetchRelated(it));
+      embed['article-ids']
```

Comment:
`article-ids` er fra det gamle embed-formatet som vil utgå. Trenger vi da denne? (Ikke bakoverkompatibel i implementeringen i `article-converter`)

Reply:
Sorry se bort i fra denne, var litt for langt frem i tankene. Denne løser endres vel i https://github.com/NDLANO/editorial-frontend/pull/200

Reply:
Nei, men det er en annen pull request som ikke er klar for master enda?